transform/operations_test.go

```diff
@@ -375,21 +400,108 @@ func TestTimeParse(t *testing.T) {
 	runOpTests(t, func() transformOperation { return &timeParse{} }, tests)
 }
 
+func TestCurrentTime(t *testing.T) {
+	tests := []opTests{
+		{
+			description: "Simple working case",
+			args:        map[string]string{"format": "RFC3339"},
+			want:        time.Now().Format(time.RFC3339),
+		},
+		{
+			description: "Missing arg",
+			args:        map[string]string{},
+			wantInitErr: true,
+		},
+		{
+			description: "Too many args",
+			args:        map[string]string{"format": "RFC3339", "cookies": "failure"},
+			wantInitErr: true,
+		},
+		{
+			description: "Format not predefined",
+			args:		 map[string]string{"format": "Mon Jan 2 15:04:05 MST 2006"},
+			want:		time.Now().Format("Mon Jan 2 15:04:05 MST 2006")	,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.description, func(t *testing.T) {
+			op, err := runOpTestInit(func() transformOperation { return &currentTime{} }, test)
+			if err != nil && test.wantInitErr {
```

Comment:
Don't you have `runOpTestInit` which can do all this error checking and stuff already?

Reply:
Working on this. I see what you mean.